src/pages/Admin/ToolsPage/ToolsPage.scss

```diff
@@ -0,0 +1,22 @@
+@import "scss/constants.scss";
+
+.ToolsPage {
+  height: 100%;
+  padding: $spacing--sm;
+
+  &__back {
+    margin-bottom: $spacing--md;
+  }
+}
+
+.Tool {
```

Comment:
If this is for a top level component, shouldn't it have it's own file in its own directory with own `index.ts`?... the works.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/2747/commits/2949bdd7d241295eb345942957244b8dfe7f6634